scripts.d/.env-win

```diff
@@ -6,11 +6,11 @@ function proxy-edf()
 }
 
 SCBI_WINDOWS_VERSION=10
-SCBI_WINDOWS_KIT_VERSION=10.0.19041.0
+SCBI_WINDOWS_KIT_VERSION=10
 SCBI_WINDOWS_KIT="C:/Program Files (x86)/Windows Kits/$SCBI_WINDOWS_VERSION"
-SCBI_WINDOWS_VC="C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133"
+SCBI_WINDOWS_VC="C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822"
 
-SCBI_SALOME_BIN="C:/opt/SALOME-9.7.0/W64"
```

Comment:
This can be removed BTW. If you look in to the script it is not used anymore. I bet this is a leftover of my initial work when using a already download/extracted archive. Since then we are using the standard mechanism via s-salome-tar and exporting in env SALOME_BIN_MODULES.